package.json

```diff
@@ -3,25 +3,28 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "test": "node src/tests/all-tests.js"
+    "test": "node src/tests/all-tests.js",
```

Comment:
I refactored `basic-tests.js` to use a `testEnvironment` and `testCaseHolder`.  Once I did this, it was trivial to just require both tests from an all-tests module.